torch/_export/exported_program.py

```diff
@@ -78,6 +76,8 @@ def __init__(
         graph_signature: ExportGraphSignature,
         call_spec: CallSpec,
         state_dict: Dict[str, Any],
+        symbol_to_range: Dict[sympy.Symbol, RangeConstraint],
+        equality_constraints: Dict[NodeDim, List[NodeDim]],
```

Comment:
two things:
1. I think there's no downstream users for equality constraints right now, and unlikely in the future, since this is mostly useful between toplevel UX and tracer, not in IR level. So we could remove this field from exported program.

2. maybe "range_constraints" is a better name for "symbol_to_range".

Reply:
No, these also turn into assertions.

Reply:
Renamed "symbol_to_range" to "range_constraints" -- the 1st comment can be dealt with in another diff related to always running AddRuntimeAssertionsPass.

Reply:
> No, these also turn into assertions.

Yeah I mean once they are turned into assertions they are kinda not used by anyone?